pkg/Microsoft.ML.Featurizers/Microsoft.ML.Featurizers.nupkgproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>Additional ML.NET featurizers for AutoML</PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML" Version="1.4.0-preview3*" />
+    <PackageReference Include="Microsoft.ML.DataView" Version="1.4.0-preview3*" />
```

Comment:
This can be removed since it is brought in transitively from referencing Microsoft.ML.